packages/socket-mode/src/logger.ts

```diff
@@ -1,26 +1,25 @@
 import { Logger, LogLevel, ConsoleLogger } from '@slack/logger';
 
-export { Logger, LogLevel } from '@slack/logger';
+export { Logger, LogLevel };
 
 let instanceCount = 0;
 
-/**
- * INTERNAL interface for getting or creating a named Logger.
- */
-export function getLogger(name: string, level: LogLevel, existingLogger?: Logger): Logger {
-  // Get a unique ID for the logger.
-  const instanceId = instanceCount;
-  instanceCount += 1;
+export default {
+  getLogger: function getLogger(name: string, level: LogLevel, existingLogger?: Logger): Logger {
```

Comment:
This restructuring allows for easier mocking without the need for a tool like `rewire` or `proxyquire`.